### DISCUSSION TITLE
Aspect ratio don't respect the height of a parent element

### DISCUSSION THREAD

**Discussion by merijnponzo** [tailwindlabs/tailwindcss#6221] [Help]
Hello,

I want to use a aspect ratio box within a parent with a fixed height, but the aspect ratio box is breaking out.
https://play.tailwindcss.com/pu4oEy6jxS

Would this be possible with the aspect ratio plugin?

**Comment by florianbouvot** [comment]
@merijnponzo I just open a request for the same issue and add a demo of how to solve it.
